product_abc_classification/views/product_view.xml

```diff
@@ -31,12 +31,12 @@
                 <field
                     name="abc_classification_profile_id"
                     optional="hide"
-                    attrs="{'invisible':['|',('type','!=','product'),('product_variant_count', '>', 1)]}"
```

Comment:
why? remember that in the read_group of _fill_data_from_stock_moves method, it has the condition `("product_id.type", "=", "product")`,

Reply:
ok